topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,428 @@
+---
+layout: tutorial_hands_on
+
+title: End-to-End Tissue Microarray Analysis with Galaxy-MTI
+questions:
+- What tools are available for pre-processing multiplex tissue images in Galaxy?
+- What tools are available for downstream analysis of multiplex tissue images in Galaxy?
+- How do I pre-process and analyze Tissue Microarray data?
+- How can I visualize multiplex tissue images and associated data?
+- How can I assign phenotypes to cells in an MTI dataset? 
+objectives:
+- Understand the tools available in Galaxy for multiplex tissue imaging analysis
+- Analyze and visualize publicly available TMA data using Galaxy
+time_estimation: 3H
+key_points:
+- Galaxy has tools and workflows that can be used to process and analyze multiplex tissue images
+- Cell feature tables produced by the Galaxy TMA workflow can be used for downstream single-cell and spatial analyses
+- There are powerful interactive visualization tools available in Galaxy that can combine the real images with associated data
+- Tissue Microarray data can be analyzed using workflows that invoke MTI tools in batch
+- Segmentation quality can vary significantly depending on features of the input image, tool used, and parameters
+contributors:
+- CameronFRWatson
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+> ### {% icon comment %} Where to find Galaxy-MTI
+>
+> Galaxy-MTI tools are currently only available on [cancer.usegalaxy.org](https://cancer.usegalaxy.org/). This is subject to change in the near future and this tutorial will be updated accordingly. 
+>
+{: .comment}
+
+Multiplex tissue images are large, multi-channel images that contain intensity data for numerous biomarkers. The methods for generating multiplex tissue images are diverse, and each method can require specialized knowledge for downstream processing and analysis. The MCMICRO ({% cite Schapiro2021 %}) pipeline was developed to process multiplex images into single-cell data, and to have the range of tools to accomodate for different imaging methods. The tools used in the MCMICRO pipeline, in addition to tools for single-cell analysis, spatial analysis, and interactive visualization are available in Galaxy to facilitate comprehensive and accessible analyses of multiplex tissue images. The MCMICRO tools available in Galaxy are capable of processing Whole Slide Images (WSI) and Tissue Microarrays (TMA). WSIs are images in which a tissue section from a single sample occupies the entire microscope slide; whereas, TMAs multiplex smaller cores from multiple samples onto a single slide. This tutorial will demonstrate how to use the Galaxy multiplex imaging tools to process and analyze publicly available TMA test data provided by MCMICRO (Figure 1.).
+
+Find a full example history [here](https://cancer.usegalaxy.org/u/watsocam/h/gtnexemplar002tma)
+
+![exemplar viv](../../images/multiplex-tissue-imaging-TMA/ex2_combined_avivator.png "Exemplar-002 TMA from MCMICRO displayed in Avivator.")
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.